Sources/TSCBasic/FileSystem.swift

```diff
@@ -122,17 +122,17 @@ public enum FileMode {
     case userWritable
     case executable
 
-    internal var setMode: (Int16) -> Int16 {
+    public func setMode(_ originalMode: Int16) -> Int16 {
```

Comment:
I wanted to pull in some of the FS-related code into SwiftPM, but that depends on this function being `public`. That was also a good opportunity to convert it from this unconventional `var`-returning-a-closure notation.